scrapy/pipelines/images.py

```diff
@@ -17,6 +17,9 @@
 from scrapy.settings import Settings
 from scrapy.utils.misc import md5sum
 from scrapy.utils.python import to_bytes
+import logging
```

Comment:
Please, move this import after `import hashlib` above.

Reply:
Sure!